app/boards/shields/qaz/qaz-layouts.dtsi

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```

Comment:
I'd either fix the date or remove the header, we don't seem to be sticking strictly to adding these.

Reply:
fixed. firmware built, but I have no method to flash at the moment so testing in studio will wait until tomorrow.